doc/lightning-addpsbtoutput.7.md

```diff
@@ -1,46 +1,104 @@
 lightning-addpsbtoutput -- Command to populate PSBT outputs from the wallet
-================================================================
+===========================================================================
 
 SYNOPSIS
 --------
 
-**addpsbtoutput** *satoshi* [*initialpsbt*] [*locktime*] [*destination*]
+**addpsbtoutput** *satoshi* [*initialpsbt*] [*locktime*] [*destination*] 
 
 DESCRIPTION
 -----------
 
-`addpsbtoutput` is a low-level RPC command which creates or modifies a PSBT
-by adding a single output of amount *satoshi*.
+Command *added* in v23.11.
 
-This is used to receive funds into the on-chain wallet interactively
-using PSBTs.
+`addpsbtoutput` is a low-level RPC command which creates or modifies a PSBT by adding a single output of amount *satoshi*.
+
+This is used to receive funds into the on-chain wallet interactively using PSBTs.
+
+- **satoshi** (msat): The satoshi value of the output. It can be a whole number, a whole number ending in *sat*, a whole number ending in *000msat*, or a number with 1 to 8 decimal places ending in *btc*.
```

Comment:
This is actually type "sat", which matters if it's a raw number... (I just checked the code!)

Reply:
Thanks, I missed it. Corrected now.